src/launch/cli/service/__init__.py

```diff
@@ -13,7 +15,9 @@ def service_group():
 
 
 service_group.add_command(build)
+service_group.add_command(test)
 service_group.add_command(clean)
 service_group.add_command(create)
 service_group.add_command(generate)
 service_group.add_command(update)
+service_group.add_command(publish)
```

Comment:
Missing EOF implies pre-commit didn't run.

Reply:
made sure that pre-commit ran!!